tool/jt.rb

```diff
@@ -107,15 +106,20 @@ def self.find_graal_javacmd_and_options
     [javacmd, vm_args.map { |arg| "-J#{arg}" } + options]
   end
 
-  def self.find_mx(graal_home)
-    sibling_mx = File.expand_path("../mx/mx", graal_home)
-    sibling_parent_mx = File.expand_path("../../mx/mx", graal_home)
-    if File.executable?(sibling_mx)
-      sibling_mx
-    elsif File.executable?(sibling_parent_mx)
-      sibling_parent_mx
+  def self.which(binary)
+    ENV["PATH"].split(File::PATH_SEPARATOR).each do |dir|
+      path = "#{dir}/#{binary}"
+      return path if File.executable? path
+    end
+    nil
+  end
+
+  def self.find_mx
```

Comment:
We should update our mxversion in suite.py if we need a more recent mx.
Then the user does it manually.
I don't want any command needing the network when building.